### DISCUSSION TITLE
Row pinning

### DISCUSSION THREAD

**Discussion by tprkdev35** [KevinVandy/material-react-table#620] [Q&A] [answered]
Are you planning to support row pinning soon? Or Is there a way around to achieve the same functionality?

**Comment by KevinVandy** [comment, marked as answer]
I have started building it a couple times, and even have a draft pr for it to TanStack Table. Maybe later this year

**Reply by tprkdev35** [reply]
Is there a simple way around I can use now?

**Reply by KevinVandy** [reply]
Let me know if you find out. It's taking me a while to cover all the edge cases with all the other features

**Comment by KevinVandy** [comment]
Row Pinning is available in Material React Table V2: https://www.material-react-table.com/docs/guides/row-pinning
